src/engraving/libmscore/arpeggio.cpp

```diff
@@ -23,14 +23,17 @@
 #include "arpeggio.h"
 
 #include <cmath>
+#include <QVector>
```

Comment:
At the moment we're trying to reduce dependencies from Qt in engraving module. Make sure that you're using STL containers here and in the further PRs

Reply:
Ok, will do. If I notice that there's an unneeded, existing Qt dependency, I assume I should convert it to a STL container?

Reply:
In my backport of this PR to 3.x that include was not needed (Neither was the cmath) and 3.x's all.h doesn't either.

Reply:
You're actually using a `QVector` in chord.cpp too bit not the include